docs/swarm/swarm-tutorial/rolling-update.md

```diff
@@ -67,6 +75,16 @@ applies the update to nodes according to the `UpdateConfig` policy:
     redis
     ```
 
+    The scheduler applies rolling updates as follows:
```

Comment:
Just realized that these steps may be happening for multiple batches in parallel because of the `--update-parallelism` flag. We should probably try to explain this, although I'm not sure about the best wording.